features/landing/components/hero/hero.tsx

```diff
@@ -0,0 +1,78 @@
+import styles from "./hero.module.scss";
+import { useGetContent } from "../../api/use-get-content";
+
+export function Hero() {
+  const { data } = useGetContent();
```

Comment:
Apart from the comment I left on the index page I doesn't seem appropriate to fetch the content inside the Hero component. In fact, the API might not even return a hero section. I'd highly recommend checking out the video solution for this tasks. There are some nuggets in there 😄